assignment-2/src/view/SearchQuestions.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import tag from "../model/tag";
+
+const SearchQuestions = ({ questions, onCreateQuestion, onViewDetails, addAnswer }) => (
+    <div>
+        <h2>Results</h2>
+        
+        <table border="1">
```

Comment:
It seems that you are reusing this table --> extract it into a separate dumb component.